scripts/jenkins-common.sh

```diff
@@ -57,13 +57,6 @@ echo "done setting up nodeenv"
 echo "node version is `node --version`"
 echo "npm version is `npm --version`"
 
-# TODO: Provide a cached node_modules/ directory for faster/smaller installs
-
-# Manage the npm cache on Jenkins.
-# (In this case, remove it. That ensures from run-to-run, it is a clean npm environment)
-echo "--> Cleaning npm cache"
-npm cache clean
```

Comment:
Shouldn't hold up the PR, but I'm curious -- why was this part removed?

Reply:
As of npm 5, it exits nonzero unless you use `--force`, since supposedly now the npm cache is self-healing. `npm cache verify` works similarly, but it might be a better idea for us to just force the clean and wipe it out entirely -- what do you think @ormsbee?

Reply:
Thanks for the explanation. I think it's fine to leave it as you already have it here. If it turns out to be problematic, we can always adjust later.